scale/job/execution/configuration/configurators.py

```diff
@@ -517,17 +496,13 @@ def _configure_regular_job(config, job_exe, job_type, system_logging_level):
         config.add_to_task('post', mount_volumes={output_mnt_name: output_vol_ro},
                            env_vars={'SYSTEM_LOGGING_LEVEL': system_logging_level})
 
+
         # Configure output directory
-        # TODO: original output dir and command arg replacement can be removed when Scale no longer supports old-style
-        # job types
-        env_vars = {'job_output_dir': SCALE_JOB_EXE_OUTPUT_PATH, 'OUTPUT_DIR': SCALE_JOB_EXE_OUTPUT_PATH}
+        env_vars = {'job_output_dir': SCALE_JOB_EXE_OUTPUT_PATH}
```

Comment:
Isn't OUTPUT_DIR the new style?

Reply:
You're right. This should have been `env_vars = {'OUTPUT_DIR': SCALE_JOB_EXE_OUTPUT_PATH}`
Good catch!